_includes/configuration/advtable-value-series.md

```diff
@@ -0,0 +1,82 @@
+## `advtable_value_series`
+{{ site.requires_5_8 }}
+
+{% if pluginname != "Advanced Tables" %}
+> **Note**: The `advtable_value_series` option requires the Advanced Tables plugin.
+{% endif %}
+
+The `advtable_value_series` option allows values series to be configured that can be used to populate cells in a table.
+
+**Type:** `Object`
+
+**Default Value:**
+ - `numbers` - a natural number series.
+ - `alpha` - a base26 letter series.
+
+```js
+{
+  numbers: {
+    update: true,
+    resizable: false,
+    generator: ...
+  },
+  alpha: {
+    update: true,
+    resizable: false,
+    generator: ...
+  },
+}
+```
+
+### Series configuration
+
+| Name | Value | Requirement | Description |
+| ---- | ----- | ----------- | ----------- |
+| update | `boolean` | Optional | default: `false` - Specifies whether the table cells that are part of the series should update after changes are made to the table. |
+| resizable | `boolean` | Optional | default: `true` - Specifies whether the table cells that are part of the series are resizable when using the resize bars. |
+| generator | `(info: GeneratorInfo, rowIndex: number, columnIndex: number) => GeneratorResult` | Required | [Usage of generator](#usageofgenerator). |
+
+#### Usage of generator
+
+The `generator` is a callback function that is used to specify how a table cell that is part of the series should be updated. The callback is passed information relating to the generator and table cell, the row index and column index of the table cell. For more details, see: [GeneratorInfo](#generatorinfo).
+The callback should return an object containing the value and optionally, the classes and attributes that should be applied on the table cell. For more details, see: [GeneratorResult](#generatorresult).
+
+##### GeneratorInfo
+
+| Name | Value | Description |
+| ---- | ----- | ----------- |
+| rowType | `'thead'`, `'tbody'` or '`tfoot'` | The section of the table cell. |
```

Comment:
```suggestion
| rowType | `'thead'`, `'tbody'` or `'tfoot'` | The section of the table cell. |
```

Reply:
Also, do we normally use the Oxford comma in these situations? From notion, it looks like marketing uses it, but I don't know about docs.